aiopg/utils.py

```diff
@@ -231,3 +233,10 @@ def __exit__(self, *args):
         coroutines._COROUTINE_TYPES += (_ContextManager,)
     except:
         pass
+
+
+# map psycopg2cffi to psycopg2, so any code
+# that imports psycopg2 will use psycopg2cffi
+if PY_IMPL == 'PyPy':
+    from psycopg2cffi import compat
+    compat.register()
```

Comment:
@jettify it import `psycopg2cffi` and makes it available under `psycopg2` name in `sys.modules` (see https://github.com/svpcom/psycopg2cffi/blob/master/psycopg2cffi/compat.py)

As long as`aiopg.utils` is imported in the main package namespace before `psycopg2`, all subsequent occurences of `psycopg2` will be mapped to already imported `psycopg2cffi` (including relative import cases that go through the main `aiopg` namespace anyway).